drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -6935,9 +6935,42 @@ uint64_t RenderingDeviceVulkan::get_captured_timestamps_frame() const {
 	return frames[frame].index;
 }
 
+static void mult64to128(uint64_t u, uint64_t v, uint64_t &h, uint64_t &l) {
```

Comment:
It might be good to avoid one-letter variables or add some comments here.

Reply:
h and l are the high and low parts of the 128 bits number, which I guess can be guessed easily from the function name. It's kind of a convention even in cpu instructions to name them like that.